### PR TITLE
Update designer.md

### PR DESCRIPTION
**Pull request by ThomDietrich**
There should be more changes. This is Mac specific without telling so and where to put the designer should be up to the user.

### CODE REVIEW

File: installation/designer.md [hunk 1]
```diff
@@ -8,26 +8,25 @@ layout: documentation
...
 The Designer is an eclipse-based IDE allowing to edit text configuration file.
 
-## Install
+## Installation
 
-To do this:
-
-* Go to the following site (it contains the latest version) : `https://github.com/eclipse/smarthome/blob/master/docs/documentation/community/downloads.md#designer-builds.`
-* In the nightly snapshots build, click on the link: `Eclipse SmartHome Designer (Mac OSX 64bit)`, to download it in your "download" folder.
+* the latest version is available for Download here: [Eclipse SmartHome Designer Downloads](https://github.com/eclipse/smarthome/blob/master/docs/documentation/community/downloads.md#designer-builds)
```
Comment (kaikreuzer) [inline installation/designer.md]: can you adapt "Download" to "download"?

File: installation/designer.md [hunk 2]
```diff
@@ -8,26 +8,25 @@ layout: documentation
...
+* get the designer for your plattform (Windows/OSX/Linux)
+* as the SmartHome Designer is still in development, consider selecting the nightly snapshots build
 
 ![](images/smarthone-designer-05.png)
 
-* with the finder, navigate to the `openHAB` folder we created during the OpenHab installation
-* create a new folder called `extra` in the `openHAB` folder
-* move the file we have just downloaded inside this folder
+* navigate to the `openHAB` folder we created during the OpenHAB installation
```
Comment (kaikreuzer) [inline installation/designer.md]: could you fix the spelling to a small 'o'?

**Comment by ThomDietrich** [comment]
Ah, what about the MacOSX and folder thing? I would suggest to make everything more general and give the user free choice to save the designer wherever he/she wants.

**Comment by kaikreuzer** [comment]
Thanks!
Yes, I agree that there should be nothing Mac specific, there is a lot room for improvement on this site.

> navigate to the openHAB folder we created during the openHAB installation

This can even be very confusing as people often install the runtime on the Raspberry Pi, but they should not try to do the same with the Designer then!
So yes, feel free to further improve the docs, I am very thankful for any update!

**Pull request merged**